src/entity/Contract.ts

```diff
@@ -242,7 +242,7 @@ export class Version extends BaseEntity {
                     logger.debug(`\tLocal Test Net Account: ${account}`);
                     driver
                         .deploy({ data: this.bytecode })
-                        .send({ from: account, gasLimit: 8000000 }, (err, tx_hash) => {
+                        .send({ from: account, gasLimit: 134217728 }, (err, tx_hash) => {
```

Comment:
This was part of our debug session for the initial network issue you had.  Please remove this change.